packages/graphback-core/tests/mockAnnotations.graphql

```diff
@@ -1,15 +1,17 @@
-type Note @create {
+
+""" @crud.delete: false """
```

Comment:
I see the annotations work as one-liners..great! Should we stick with one format though, at least with our examples/tests and templates? We can document both ways to do it, but stick with one of ourself, for nothing other than consistency really.

Reply:
Which one you preffer. 
I check spec and there is no official format. Which one you like more

Reply:
I so see value in showcasing both as not everyone will be able to see if both are possible :D

Reply:
I find the multiline annotation easier to read because the annotation part has its own newline. If you see examples 2 and 3 below, single-line annotations may become even harder to read when we arguments are introduced.

```gql
"""
@crud.foo: 'bar'
"""
propOne: String
```

```gql
"""@crud.foo: 'bar'"""
propTwo: String
```

```gql
""" @crud.foo: 'bar' """
propTwo: String
```